pytools/graph.py

```diff
@@ -410,6 +411,62 @@ def compute_induced_subgraph(graph: Mapping[NodeT, Set[NodeT]],
 # }}}
 
 
+# {{{ get_graph_dot_code
+
+def get_graph_dot_code(graph: Mapping[T, Collection[T]],
```

Comment:
```suggestion
def as_graphviz_dot(graph: GraphT[NodeT],
    node_labels: Optional[Callable[[NodeT], str]] = None,  # default str?
    edge_labels: Optional[Callable[[NodeT, NodeT], str]] = None)
```

Reply:
done in [d84cb73](https://github.com/inducer/pytools/pull/151/commits/d84cb7397ec007d3b6913c25fe16e3fc1fea3682)